pkgs/servers/sql/postgresql/ext/timescaledb.nix

```diff
@@ -8,17 +8,17 @@
 
 stdenv.mkDerivation rec {
   pname = "timescaledb";
-  version = "2.5.2";
+  version = "2.6.0";
 
   nativeBuildInputs = [ cmake ];
   buildInputs = [ postgresql openssl libkrb5 ];
 
   src = fetchFromGitHub {
-    owner  = "timescale";
-    repo   = "timescaledb";
+    owner = "timescale";
+    repo = "timescaledb";
     # some branches are named like tags which confuses git
-    rev    = "refs/tags/${version}";
-    sha256 = "sha256-eq2dljS8+0fRcA7hQr5yOV1FXtMyaajjYFqpw+DYt1A=";
+    rev = "refs/tags/${version}";
```

Comment:
doesn't `rev = version;` work the same?

Reply:
Sorry about that! I merged this before my morning coffee
>doesn't rev = version; work the same?

yes, it works the same. I'll address it in the next bump.

Reply:
no problem, and sorry for being so confrontational about it.

there seem to be a bunch (around 100) of packages using `refs/tags`, maybe we should change them all, assuming there is actually no good reason to do that.
also the documentation (doc/builders/fetchers.chapter.md) should maybe be rephrased.

Reply:
@ajs124 sorry for not responding in the first place! I was waiting on further comments :) ...I'll change it in the next update (if it doesn't get updated in the meantime with the other pkgs).